.erda/migrations/core-services/20220713-erda_rule_set-base.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS `erda_rule_sets` (
```

Comment:
according to our sql standard:
```suggestion
CREATE TABLE IF NOT EXISTS `erda_rule_set` (
```

Reply:
done